README.md

```diff
@@ -67,12 +67,12 @@ clusterctl init --infrastructure=packet
 To generate your cluster yaml:
 
 1. Set the required environment variables:
-   * `PACKET_PROJECT_ID` - Packet project ID
```

Comment:
I have two concerns related to changing these from `PACKET_PROJECT_ID` to `PROJECT_ID` and `PACKET_FACILITY` to `FACILITY`.

* we use those env vars in other places as well, e.g. packet-ccm and csi-packet, at a minimum
* most cloud providers prepend env vars with their name or short form, so that they don't clash with more generic things. Think `AWS_ACCESS_KEY_ID` or `AZURE_CLIENT_ID` or `GOOGLE_APPLICATION_CREDENTIALS` or `DO_PAT`. I don't think packet should be different.

Reply:
I was mainly trying to align with what is used in the existing quick start, but I do agree that we should prefix with `PACKET_`, so we may need to coordinate this change along with:
- An updated release
- An update to the CAPI quickstart docs

Reply:
The quick start doesn't have `PACKET_`? Oh oops. They should. 😄

Reply:
+1 for `PACKET_`, I'd like to see all Packet tools benefit from a common set of environment variables and eventually a common config file format.

Reply:
This thread didn't age well.

Reply:
https://github.com/packethost/packngo/issues/189 would be very helpful here.